openvpn/middlewares/server/auth/authenticator.go

```diff
@@ -0,0 +1,16 @@
+package auth
+
+// AuthenticatorChecker callback checks given auth primitives (i.e. customer identity signature / node's sessionId)
+type AuthenticatorChecker func(username, password string) (bool, error)
+
+// NewAuthenticatorFake returns AuthenticatorChecker callback
+func NewAuthenticatorFake() AuthenticatorChecker {
```

Comment:
Dont repeat package name in subnames:
- `authenticator.go` -> `checker.go`
- `NewAuthenticatorFake()` -> NewCheckerFake()